ops/roles/cgus/tasks/database.yml

```diff
@@ -0,0 +1,44 @@
+- name: Remove index.lock in {{ name }}
+  file:
+    path: "/home/{{ ansible_user }}/{{ directory }}/.git/index.lock"
+    state: absent
+
+- name: Create backup tag for {{ name }}
```

Comment:
Now that we simply `reset`, we don't really need these tag backups _if they are cleaned up automatically_: if we have strong confidence in our process, we can always resuscitate history with `reflog`. These are not eternal, but I'd expect them to survive them before `git gc` runs for about as long —if not longer— than 20 deployments.